src/twisted/conch/interfaces.py

```diff
@@ -18,30 +18,36 @@ class IConchUser(Interface):
     def lookupChannel(channelType, windowSize, maxPacket, data):
         """
         The other side requested a channel of some sort.
-        channelType is the type of channel being requested,
-        windowSize is the initial size of the remote window,
-        maxPacket is the largest packet we should send,
-        data is any other packet data (often nothing).
+
+        C{channelType} is the type of channel being requested,
+        as an ssh connection protocol channel type.
+        C{data} is any other packet data (often nothing).
 
         We return a subclass of L{SSHChannel<ssh.channel.SSHChannel>}.  If
         an appropriate channel can not be found, an exception will be
-        raised.  If a L{ConchError<error.ConchError>} is raised, the .value
-        will be the message, and the .data will be the error code.
+        raised.  If a L{ConchError<error.ConchError>} is raised, the C{.value}
+        will be the message, and the C{.data} will be the error code.
 
-        @type channelType:  L{str}
+        @param channelType: The requested channel type
+        @type channelType:  L{bytes}
```

Comment:
Whoops!  This certainly [appears to be bytes](https://github.com/twisted/twisted/blob/91a72d075db94a78da07d8d601fad090d91d5504/src/twisted/conch/ssh/connection.py#L136).  Thanks for fixing the type here!